base_layer/core/src/base_node/tari_pulse_service/mod.rs

```diff
@@ -329,9 +332,22 @@ async fn check_health(
     notify_comms_health: watch::Sender<Vec<LivenessCheckResult>>,
 ) {
     let results = Arc::new(RwLock::new(Vec::new()));
-    let peers = node_comms.get_seeds().await.unwrap_or_else(|_| vec![]);
+
+    let peers = match tokio::time::timeout(Duration::from_secs(10), node_comms.get_seeds()).await {
+        Ok(Ok(seeds)) => seeds,
+        Ok(Err(err)) => {
+            warn!(target: LOG_TARGET, "check_health: failed to get seed peers: {}", err);
+            return;
```

Comment:
previously on fail we returned empty array of peers `unwrap_or_else(|_| vec![]);`
but here we are early exiting the function. We keep the old behavior.
```suggestion
            vec![]
 ```

Reply:
I think this is fine, lets not suppress the error